sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/api/Azure.ResourceManager.ConfidentialLedger.netstandard2.0.cs

```diff
@@ -51,6 +57,46 @@ public partial class ConfidentialLedgerResource : Azure.ResourceManager.ArmResou
         public virtual Azure.ResourceManager.ArmOperation<Azure.ResourceManager.ConfidentialLedger.ConfidentialLedgerResource> Update(Azure.WaitUntil waitUntil, Azure.ResourceManager.ConfidentialLedger.ConfidentialLedgerData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.ConfidentialLedger.ConfidentialLedgerResource>> UpdateAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.ConfidentialLedger.ConfidentialLedgerData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
+    public partial class ManagedCCFCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.ConfidentialLedger.ManagedCCFResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.ConfidentialLedger.ManagedCCFResource>, System.Collections.IEnumerable
```

Comment:
Consider to use `prepend-rp-prefix` for `ManagedCCF` as well

Reply:
ManagedCCF is a very big module which goes parallelly with ConfidentialLedger. It wont be feasible to change this class name as several customers will be looking for that name specifically.